src/plugins/chart_expressions/expression_metric/common/expression_functions/__snapshots__/metric_function.test.ts.snap

```diff
@@ -22,7 +22,7 @@ Object {
 
 exports[`interpreter/functions#metric returns an object with the correct structure 1`] = `
 Object {
-  "as": "metric_vis",
+  "as": "metricVis",
```

Comment:
Why this change?

Reply:
https://github.com/elastic/kibana/blob/f38230fe5241104e97bf1afe1f1462fe829d2161/src/plugins/vis_types/metric/public/to_ast.ts#L65

Before at to_ast, you were rendering expression with name `metricVis`, but the expression function was `metric_vis`. Right now they have the same names, as they should.

Reply:
Hm, I don't have a strong opinion about this, I think we should stick to a naming scheme though.
So far all of the renderers are using snake case:
<img width="399" alt="Screenshot 2021-09-30 at 16 38 16" src="https://user-images.githubusercontent.com/1508364/135476582-2ed4a83b-0aa6-40df-b120-defc0d7729ea.png">

but OTOH the "regular" canvas functions are using camel case (except for `var_set` which is inconsistent as well):
<img width="154" alt="Screenshot 2021-09-30 at 16 40 00" src="https://user-images.githubusercontent.com/1508364/135476683-868ed97a-c82d-463e-bac1-73e3bbd640e1.png">


@ppisljar could we get guidance here? Which naming scheme should be used for expression function names and renderers?

Reply:
i think we want to go camelCase direction. no strong opinion, its just what we've been moving towards on appservices team.

Reply:
Sounds good to me, we can rename the Lens stuff once we start moving it in